libtransmission/session.h

```diff
@@ -222,6 +222,9 @@ struct tr_session
 
     struct tr_bindinfo* public_ipv4;
     struct tr_bindinfo* public_ipv6;
+
+    struct tr_list* defaultTrackers;
+    char* defaultTrackersStr;
```

Comment:
These are leaked; need to be cleaned up at the end of `tr_sessionClose()` in `session.c`

Reply:
Done.